msteams-platform/tabs/how-to/app-caching.md

```diff
@@ -77,9 +77,82 @@ microsoftTeams.teamsCore.registerBeforeUnloadHandler((readyToUnload) => {
+
+> [!NOTE]
+> The debug tool for cached apps is available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+You can enable the Proto Task Manager in Teams, a debug tool that shows the status of your cached apps. In your Teams client, select the **Control+Shift+Alt+8** keys on Windows or **Command+Shift+Option+8** on Mac to open the Proto Task Manager.
+
+:::image type="content" source="../../assets/images/tabs/app-cache-debug-tool.png" alt-text="Screenshot shows the caching tab in the Proto Task Manager in Teams." lightbox="../../assets/images/tabs/app-cache-debug-tool.png":::
+
+The **AppCaching** tab contains the following details:
+
+* **state**: Shows the app's cached or uncached state.
+* **isActive**: Shows the active or inactive status of the cached app.
+* **timeElapsed**: Shows the time elapsed since the app was cached.
+* **supportsLoad**: Shows if the app has registered the `Load` handler if app caching is enabled.
+* **supportsBeforeUnload**: Shows if the app has registered the `BeforeUnload` handler if app caching is enabled.
+* **totalFrameMemory**: Shows the memory usage of the app.
+* **totalFrameCommitMemory**: Shows the CPU usage of the app.
+
+## Precaching tab apps
+
+> [!NOTE]
+>
+> Precaching tab apps is available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+While caching reduces the subsequent load times of an app, precaching optimizes an app's initial load time by allowing Teams to preload the app. Teams preloads apps in the background after launch or when idle, based on users' recent app usage patterns and the apps' cache history. The preloaded apps remain cached until the user opens the app, resulting in a faster loading time.
+
+### Enable precaching for tab app
+
+To enable precaching for your tab app, follow these steps:
+
+1. [Enable app caching](#enable-app-caching).
+
+1. Update your app manifest as follows:
+
+    1. Set the value of `showLoadingIndicator` to `true`. This action ensures that Teams waits until your app sends `notifySuccess` to conclude the app load sequence during precaching. For more information, see [showLoadingIndicator](../../resources/schema/manifest-schema-dev-preview.md#showloadingindicator).
+
+    1. Add the `backgroundLoadConfiguration` object and define the `contentUrl`.
+
+        ```json
+        {
+            //...
+            "staticTabs": [
+            {
+                "contentUrl": "https://www.contoso.com/content?host=msteams"
+            }
+        ],
+        "backgroundLoadConfiguration": {
+            "tabConfiguration": {
+                "contentUrl": "https://www.contoso.com/content?host=msteams"
+                }
+            }
+            //...
+        }
+        ```
+
+        > [!NOTE]
+        > The `contentUrl` shouldn't contain context-specific parameters, such as team site URL or thread ID, as Teams loads apps with no prior context during launch.
```

Comment:
Add some more tips:
This contentUrl should be generic enough to load in the background without any user interaction.